static/app/views/dashboards/widgetBuilder/widgetBuilder.tsx

```diff
@@ -967,17 +969,25 @@ function WidgetBuilder({
                     <Layout.Page>
                       <Header
                         orgSlug={orgSlug}
-                        title={state.title}
                         dashboardTitle={dashboard.title}
                         goBackLocation={previousLocation}
-                        onChangeTitle={newTitle => {
-                          handleDisplayTypeOrTitleChange('title', newTitle);
-                        }}
                       />
                       <Body>
                         <MainWrapper>
                           <Main>
                             <BuildSteps symbol="colored-numeric">
+                              <BuildStep title={t('Name your widget')}>
+                                <TitleInput
+                                  name="title"
```

Comment:
```suggestion
                                  name="title"
                                  inline={false}
```

This removes the nested padding left from the input field